app/scripts/lib/bloomfilter.js

```diff
@@ -0,0 +1,118 @@
+define([
+],
+function() {
+  'use strict';
+  var typedArrays = typeof ArrayBuffer !== "undefined";
```

Comment:
If we want to pull this out, ISTM the right factoring here is "fxa-password-strength-checker" and we expose an API designed for doing that, leaving the fact that it uses a bloom filter as an implementation detail.  @TDA thoughts?  I'm happy to create you a separate repo, ping me async and we'll coordinate.